Userland/Libraries/LibTLS/TLSv12.h

```diff
@@ -241,7 +241,7 @@ struct Options {
         { HashAlgorithm::SHA384, SignatureAlgorithm::RSA },
         { HashAlgorithm::SHA256, SignatureAlgorithm::RSA },
         { HashAlgorithm::SHA1, SignatureAlgorithm::RSA });
-    OPTION_WITH_DEFAULTS(Vector<NamedCurve>, elliptic_curves, NamedCurve::x25519)
+    OPTION_WITH_DEFAULTS(Vector<NamedCurve>, elliptic_curves, NamedCurve::x25519, NamedCurve::x448)
```

Comment:
Can we split this into two lines now? not a big fan of these lists growing on one line
```suggestion
    OPTION_WITH_DEFAULTS(Vector<NamedCurve>, elliptic_curves,
        NamedCurve::x25519,
        NamedCurve::x448)
```